pgcli/packages/pgspecial/main.py

```diff
@@ -0,0 +1,108 @@
+import logging
```

Comment:
These two imports are used. The logging module is used to initialize logging which is used to log a message in the `register_special_command()` and `from . import export` is used to import the decorator called `@export` which is also used in the code.